Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -174,50 +181,58 @@ fun Home(
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun HomeAppBar(
-    backgroundColor: Color,
-    modifier: Modifier = Modifier
+private fun HomeAppBar(
+    selectedHomeCategory: HomeCategory,
+    homeCategories: List<HomeCategory>,
+    onHomeCategorySelected: (HomeCategory) -> Unit,
+    modifier: Modifier = Modifier,
+    showHomeCategoryToggle: Boolean = false,
 ) {
     TopAppBar(
         title = {
-            Row {
-                Image(
-                    painter = painterResource(R.drawable.ic_logo),
-                    contentDescription = null
-                )
-                Icon(
-                    painter = painterResource(R.drawable.ic_text_logo),
-                    contentDescription = stringResource(R.string.app_name),
-                    modifier = Modifier
-                        .padding(start = 4.dp)
-                        .heightIn(max = 24.dp)
-                )
-            }
-        },
-        actions = {
-            IconButton(
-                onClick = { /* TODO: Open search */ }
-            ) {
-                Icon(
-                    imageVector = Icons.Filled.Search,
-                    contentDescription = stringResource(R.string.cd_search)
-                )
-            }
-            IconButton(
-                onClick = { /* TODO: Open account? */ }
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(end = 16.dp)
             ) {
-                Icon(
-                    imageVector = Icons.Default.AccountCircle,
-                    contentDescription = stringResource(R.string.cd_account)
-                )
+                if (showHomeCategoryToggle) {
+                    HomeCategoryTabs(
+                        categories = homeCategories,
+                        selectedCategory = selectedHomeCategory,
+                        onCategorySelected = onHomeCategorySelected,
+                        modifier = Modifier.width(240.dp),
+                        showHorizontalLine = false
+                    )
+                    Spacer(modifier = Modifier.weight(1f))
+                }
+                SearchBar(
+                    query = "Jetcaster",
```

Comment:
Bit of a weird placeholder search query to have the app name

Reply:
Agreed. I changed this to a proper placeholder text which showing the text "Search for a podcast"